e2e/api/dataHelper.js

```diff
@@ -14,18 +14,34 @@ const getDate = days => {
   return date;
 };
 
+
 module.exports = {
   date: (days = 0) => {
     return getDateTimeISOString(days).slice(0, 10);
   },
 
-  dateNoWeekends: function dateNoWeekends(days = 0) {
-    let date = getDate(days);
-
-    if(date.getDay() !== 6 && date.getDay() !== 0) {
-      return date.toISOString().slice(0, 10);
+  dateNoWeekends: async function dateNoWeekends(days = 0) {
+    const date = getDate(days);
+    let date_String = date.toISOString().slice(0, 10);
+    let isDateABankHoliday = false;
+    if (date.getDay() !== 6 && date.getDay() !== 0) {
+      return date_String;
     } else {
```

Comment:
No need for the `else` since the previous statement is a `return`.

I would suggest to just close the brace and that's it.

Short-circuiting code improves readability dramatically and can prevent mistakes (like putting a closing brace on the wrong line causing some following code to never be executed even though it should: usually happens in merges with conflicts).

Reply:
This code has been updated.